python/strings_udf/strings_udf/_lib/cpp/strings_udf.pxd

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.
+
+from libc.stdint cimport uint8_t
+from libcpp.memory cimport unique_ptr
+from libcpp.string cimport string
+from libcpp.vector cimport vector
+
+from cudf._lib.cpp.column.column cimport column
+from cudf._lib.cpp.column.column_view cimport column_view
+from cudf._lib.cpp.types cimport size_type
+from rmm._lib.device_buffer cimport DeviceBuffer, device_buffer
+
+
+cdef extern from "cudf/strings/udf/udf_apis.hpp" nogil:
+    cdef unique_ptr[device_buffer] to_string_view_array(column_view)
```

Comment:
Unless this function should be noexcept (it's not currently, but maybe it should be?) this needs an `except +`.